Dockerfile

```diff
@@ -16,19 +16,32 @@ RUN apt-get -y update
 
 #-------------Application Specific Stuff ----------------------------------------------------
 
-RUN apt-get -y install unzip openjdk-7-jre-headless openjdk-7-jre
+EXPOSE 8080
+
+ENV GS_VERSION 2.6.1
+ENV GEOSERVER_DATA_DIR /opt/geoserver/data_dir
 
 ADD resources /tmp/resources
 
-# A little logic that will fetch the geoserver zip file if it
-# is not available locally in the resources dir and
+# A little logic that will fetch the geoserver war zip file if it
+# is not available locally in the resources dir
 RUN if [ ! -f /tmp/resources/geoserver.zip ]; then \
-    wget -c http://downloads.sourceforge.net/project/geoserver/GeoServer/2.6.1/geoserver-2.6.1-bin.zip -O /tmp/resources/geoserver.zip; \
+      wget -c http://downloads.sourceforge.net/project/geoserver/GeoServer/${GS_VERSION}/geoserver-${GS_VERSION}-war.zip \
+	-O /tmp/resources/geoserver.zip; \
     fi; \
-    unzip /tmp/resources/geoserver.zip -d /opt && mv -v /opt/geoserver* /opt/geoserver
-ENV GEOSERVER_HOME /opt/geoserver
-ENV JAVA_HOME /usr/
-
-#ENTRYPOINT "/opt/geoserver/bin/startup.sh"
-CMD "/opt/geoserver/bin/startup.sh"
-EXPOSE 8080
+    unzip /tmp/resources/geoserver.zip -d /tmp/geoserver \
+    && mv /tmp/geoserver/geoserver.war /usr/local/tomcat/webapps/geoserver.war \
+    && unzip /usr/local/tomcat/webapps/geoserver.war -d /usr/local/tomcat/webapps/geoserver \
```

Comment:
The manual war unzipping is done here so that jars from the plugin zips can be added to GeoServer's `WEB-INF/lib` directory. Otherwise the GeoServer WAR would have to be unzipped to a temp directory, the plugin jars added there, and the WAR rezipped only to be exploded by Tomcat again. Seemed better to just do it this way.